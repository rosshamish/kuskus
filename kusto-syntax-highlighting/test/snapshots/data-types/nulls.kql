print bool(null), datetime(null), dynamic(null), guid(null), int(null), long(null), real(null), time(null)
