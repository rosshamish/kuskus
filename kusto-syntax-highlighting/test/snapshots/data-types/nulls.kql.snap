>print bool(null), datetime(null), dynamic(null), guid(null), int(null), long(null), real(null), time(null)
#^^^^^ source.kusto keyword.other.query.kusto
#     ^ source.kusto
#      ^^^^ source.kusto storage.type.kusto
#          ^ source.kusto
#           ^^^^ source.kusto constant.language.kusto
#               ^^^ source.kusto
#                  ^^^^^^^^ source.kusto storage.type.kusto
#                          ^ source.kusto
#                           ^^^^ source.kusto constant.language.kusto
#                               ^^^ source.kusto
#                                  ^^^^^^^ source.kusto storage.type.kusto
#                                         ^ source.kusto
#                                          ^^^^ source.kusto constant.language.kusto
#                                              ^^^ source.kusto
#                                                 ^^^^ source.kusto storage.type.kusto
#                                                     ^ source.kusto
#                                                      ^^^^ source.kusto constant.language.kusto
#                                                          ^^^ source.kusto
#                                                             ^^^ source.kusto storage.type.kusto
#                                                                ^ source.kusto
#                                                                 ^^^^ source.kusto constant.language.kusto
#                                                                     ^^^ source.kusto
#                                                                        ^^^^ source.kusto storage.type.kusto
#                                                                            ^ source.kusto
#                                                                             ^^^^ source.kusto constant.language.kusto
#                                                                                 ^^^ source.kusto
#                                                                                    ^^^^ source.kusto storage.type.kusto
#                                                                                        ^ source.kusto
#                                                                                         ^^^^ source.kusto constant.language.kusto
#                                                                                             ^^^ source.kusto
#                                                                                                ^^^^ source.kusto storage.type.kusto
#                                                                                                    ^ source.kusto
#                                                                                                     ^^^^ source.kusto constant.language.kusto
#                                                                                                         ^^ source.kusto
>